tests/integration/api/v1/sponsored-beta/get.test.js

```diff
@@ -0,0 +1,111 @@
+import { defaultTabCashForAdCreation } from 'tests/constants-for-tests';
+import orchestrator from 'tests/orchestrator.js';
+import RequestBuilder from 'tests/request-builder';
+
+beforeAll(async () => {
+  await orchestrator.waitForAllServices();
+});
+
+describe('GET /api/v1/sponsored-beta', () => {
+  describe('Anonymous user', () => {
+    const adsRequestBuilder = new RequestBuilder('/api/v1/sponsored-beta');
+    let owner;
+
+    beforeEach(async () => {
+      await orchestrator.dropAllTables();
+      await orchestrator.runPendingMigrations();
+      owner = await orchestrator.createUser();
+
+      orchestrator.createBalance({
+        balanceType: 'user:tabcash',
+        recipientId: owner.id,
+        amount: 100 * defaultTabCashForAdCreation,
+      });
+    });
+
+    it('should never get default content', async () => {
+      await orchestrator.createContent({
+        owner_id: owner.id,
+        title: 'Content',
+        status: 'published',
+        type: 'content',
+      });
+
+      const { response, responseBody } = await adsRequestBuilder.get();
+
+      expect.soft(response.status).toEqual(200);
+      expect(responseBody).toEqual([]);
+    });
+
+    it('should never get unpublished ad', async () => {
+      await orchestrator.createContent({
+        owner_id: owner.id,
+        title: 'Draft Ad',
+        status: 'draft',
+        type: 'ad',
+      });
+
+      const deletedAd = await orchestrator.createContent({
+        owner_id: owner.id,
+        title: 'Deleted Ad',
+        status: 'published',
+        type: 'ad',
+      });
+
+      await orchestrator.updateContent(deletedAd.id, { status: 'deleted' });
+
+      const { response, responseBody } = await adsRequestBuilder.get();
+
+      expect(response.status).toEqual(200);
+      expect(responseBody).toEqual([]);
+    });
+
+    it('should limit the number of ads returned', async () => {
+      const createdAds = await createAds(4, owner);
+
+      const { response, responseBody } = await adsRequestBuilder.get();
+
+      expect.soft(response.status).toEqual(200);
```

Comment:
Aproveitando que provavelmente terá alterações no código, pode usar `toBe` aqui e nos outros testes:

```suggestion
      expect.soft(response.status).toBe(200);
```